biz.aQute.bndlib/bnd.bnd

```diff
@@ -38,6 +37,7 @@ Export-Package: \
 	aQute.bnd.util.home;-noimport:=true,\
 	aQute.bnd.util.repository;-noimport:=true,\
 	aQute.bnd.component.error;-noimport:=true,\
+	org.osgi.annotation.bundle,\
```

Comment:
bndlib should not contain or export this package. bndlib should not have any runtime dependency on this package.

Reply:
so should it forcibly do `!org.osgi.annotation.bundle` on import? The annotations packages USE the enums in `org.osgi.annotation.bundle.Requirement`! Which makes them part of the required API.

Reply:
We should not even put that package in the jar. This is for the same reason we no longer put the OSGi ds and metatype annotation packages in the jar. It created problem in non-OSGi scenarios where bndlib is on the classpath with the OSGi annotation jars and people end up with the wrong classes. Since we don't want to end up with similar issues with another OSGi annotation package, we must not put this package in the bnd jars. The answer may be to not use the OSGi enum types in the Bnd annotations but add specific support, like we do for the OSGi annotations, to map the UPPERCASE enum names to lowercase for the generated manifest headers.

Reply:
See https://github.com/bndtools/bnd/pull/2637/commits/e6470d48133a0f3305347e52f77149686fe98325 where we changed to stop using the real OSGi bundle annotations in bndlib.